.github/workflows/test_head_and_bump.yaml

```diff
@@ -36,10 +36,13 @@ jobs:
         run: |
           python build_tools/update_pinned.py
           cat version_info.json
-
-      - name: Push updated versions
-        run: |
           git config --local user.name "Bump Pinned Versions"
           git config --local user.email "iree-github-actions-bot@google.com"
           git commit -am "Update pinned versions"
-          git push origin update-versions
```

Comment:
I really recommend just using `git push` instead. The push action doesn't really add much other than a whole layer of nodejs indirection. See https://github.com/iree-org/iree/pull/9520. If the thing you need to do here is force, just add `--force`. Do you really want to be always force-pushing over this though? You'll lose the previous history. One option is to instead merge from main into the branch, so fetch that as well.

Reply:
Changed to a direct push. Losing the history is a tradeoff, either way the last update commit will be overwritten which seems fine.

Reply:
Yeah force push just always scares me. At least we've got protection on the main branch, so it can't accidentally override that